merlin-server/sql/merlin/functions/hasura/snapshot_functions.sql

```diff
@@ -0,0 +1,44 @@
+create table hasura_functions.create_snapshot_return_value(snapshot_id integer);
```

Comment:
Are these intermediate tables necessary for hasura functions? I.e we can't just return `_snapshot_id` as an integer?

Reply:
Yes, very sadly. [See the return type of `Supported SQL Functions` in the docs.](https://hasura.io/docs/latest/schema/postgres/custom-functions/)